clusterctl/examples/openstack/machines.yaml.template

```diff
@@ -0,0 +1,53 @@
+items:
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: openstack-master-
+    labels:
+      set: master
+  spec:
+    providerConfig:
+      value:
+        apiVersion: "openstackproviderconfig/v1alpha1"
+        kind: "OpenstackProviderConfig"
+        name: master
+        flavor: m1.medium
+        image: CC-Ubuntu16.04-20160610
+        availability_zone: nova
+        networks:
+        - uuid: ab14ce0d-5e1f-4e32-bf65-00416e3cc19c
```

Comment:
which network is this? is this the "external" network on which we have the floating ip?

Reply:
Not have to be the "external" network, just the network you want the new machine be attached.